docs/index.md

```diff
@@ -0,0 +1,301 @@
+# Granite Framework
+
+Granite is a business actions architecture for Rails apps.
+
+It's a combination of user interaction (attributes and validations), context (preconditions) and
+permissions (authorization policies).
+
+## What problems it solves?
+
+When your application becomes more prominent, you use patterns to allow you to
+leverage your productivity. Instead of start inflating the controller and
```

Comment:
You usually "leverage <something> for <something>". In this case, I would say 'leverage patterns for (maintaining/improving?) productivity'.